src/default_panels/StyleTracesPanel.js

```diff
@@ -41,7 +51,14 @@ const StyleTracesPanel = ({localize: _}) => (
     />
     <NumericFraction label={_('Opacity')} attr="opacity" />
     <ColorPicker label={_('Color')} attr="color" />
-
+    <Numeric
+      showSlider
+      min={0}
+      max={1}
```

Comment:
We shouldn't need to include `min` or `max` right? That comes from the Plotly Schema and is already baked into the framework in `unpackPlotProps`. Or it was at one time.

Reply:
Actually I am pretty sure it still is I think there is a test covering that behaviour